exercises/practice/zebra-puzzle/.meta/proof.ci.js

```diff
@@ -0,0 +1,4 @@
+export const zebraPuzzle = () => ({
+  waterDriker: 'Norwegian',
+  zebraOwner: 'Japanese',
+});
```

Comment:
This file should contain an actual in-code solution to the problem to demonstrate it can be done.

Also the instructions.append.md file should inform the student that this exercise is about writing a problem to solve the problem, not just about solving the puzzle some other way. (Example here: https://github.com/exercism/go/blob/main/exercises/practice/zebra-puzzle/.docs/instructions.append.md)